src/screens/Home/Home.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, Image, Text } from 'grommet'
+import Kelp from 'images/kelp.png'
+import Title from 'images/title.svg'
+import ResponsiveImage from 'components/ResponsiveImage'
+import styled from 'styled-components'
+import ChooseLocation from './components/ChooseLocation'
+import { ZooniverseLogotype } from '@zooniverse/react-components'
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+export default function Home() {
+  return (
+    <Box background='sand' direction='row'>
```

Comment:
I think this needs a `fill` added to it too to make sure it's taking the full viewport height.

Reply:
I think the issue here is that the Kelp image should be set as a div background. Currently, the image is rather large, which ends up stretching the image and the parent div making it the tallest div on the page. In turn, this content doesn't look tall enough.

Reply:
I think this is still needed. Before without `fill` prop on my screen:

<img width="1438" alt="Screen Shot 2020-07-21 at 11 42 06 AM" src="https://user-images.githubusercontent.com/5016731/88082316-4214d700-cb47-11ea-874c-9197524e6e20.png">

After:
<img width="1440" alt="Screen Shot 2020-07-21 at 11 42 19 AM" src="https://user-images.githubusercontent.com/5016731/88082343-4c36d580-cb47-11ea-8710-b805f489ffa1.png">

It adds the 100vh units to the div so it's always the full height of the viewport.

Reply:
@wgranger can the line next to the text on the right also extend down to the bottom of the screen?

Reply:
Weird. I'm seeing the opposite. A white space is added at the bottom only when I add the `fill` prop. Is that screenshot from Chrome?

@beckyrother Yes, that is changed with a recent commit

Reply:
Ahh, I see. This is with a larger screen. Looking into this now.